src/TheThingsNetwork.cpp

```diff
@@ -178,7 +174,13 @@ int TheThingsNetwork::sendBytes(const byte* payload, int length, int port, bool
     return -1;
   }
 
-  String response = readLine();
+  String response = "";
+  if (confirm) {
+    while ((response = readLine()) == "");
```

Comment:
readLine no longer returns empty. Get rid of this if case and merge line 177 and 182